terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars

```diff
@@ -0,0 +1,6 @@
+environment = "cidev"
+aws_profile = "development-eu-west-2"
+
+# service configs
+use_set_environment_files = true
+log_level = "trace"
```

Comment:
can remove log_level from all profiles as we will just define this in the env vars